ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/mpls.yml

```diff
@@ -7,6 +7,45 @@ mpls:
     transport_address_interface: Loopback0
   icmp:
     fragmentation_needed_tunneling: true
+  rsvp:
+    hello:
+      interval: 30
+      multiplier: 254
+    authentication:
+      password_indexes:
+        - index: 766
+          password_type: 7
+          password: SiMpLePaSs
+        - index: 55
+          password_type: '7'
+          password: SiMpLePaSsInDeX
```

Comment:
```suggestion
          password: 13361E3F1B20011A2A173B
        - index: 55
          password_type: '7'
          password: 14241B062114062E142900260B1D23133B
```

Reply:
Adding type 7 password forms of give passwords.

Reply:
```
s1-leaf1(config-mpls-rsvp)#   authentication index 55 password 7 SiMpLePaSsInDeX
% Invalid input
s1-leaf1(config-mpls-rsvp)#   authentication index 766 password 7 SiMpLePaSs
% Invalid input
```
simple password with password type 7 does not work.

Reply:
Thanks.. Fixed!